liferay-echo/liferay-portal
# LPS-180585 Add description to RSS Cache template setting

Motivation
=======
Currently, the RSS cache configuration display is not descriptive enough (specifically the Feed Time filed) for our users so, a FR was raised in order to improve it.

[Story](https://issues.liferay.com/browse/LPS-180578)
[Feature Request](https://issues.liferay.com/browse/LPS-155391)

Proposed Solution
========
Add a description under the Feed Time field.

Steps to Verify
========

1. Go to System Settings > RSS > Cache
2. The "Set the time interval in which the information of the cache is renewed." description should be shown under the Feed Time imput.

Screenshots 
========
<img width="1266" alt="RSS-Cache-Result" src="https://user-images.githubusercontent.com/85171101/229518087-e0c93719-44fe-4ebf-8f29-8da3da2a8b45.png">

Commits
========

- LPS-180585 Adding feed time help message
- LPS-180585 buildLang
- LPS-180585 Adding description help message to RSS Cache configuration
